grug/types/src/coins.rs

```diff
@@ -516,4 +554,45 @@ mod tests {
         let s = "uatom:123,uatom:456";
         assert!(Coins::from_str(s).is_err())
     }
+
+    #[test]
+    fn saturating_deduct_many() {
+        // Deduct less than available
+        let mut coins = mock_coins();
+        let deduct = Coins::try_from([
+            ("uatom", Uint128::new(100)),
+            ("umars", Uint128::new(100)),
+            ("uosmo", Uint128::new(789)),
+        ])
```

Comment:
For tests, instead of `Coins::try_from` we can simply do `Coins::new_unchecked`.